packages/expo-constants/ios/EXConstantsService.m

```diff
@@ -61,15 +61,15 @@ - (NSDictionary *)constants
            @"debugMode": @(isDebugXCodeScheme),
            @"isHeadless": @(NO),
            @"nativeAppVersion": [self appVersion],
-           @"nativeBuildVersion": [self buildVersion],
+           @"nativeBuildVersion": EXNullIfNil([self buildVersion]),
            @"installationId": [_installationIdProvider getOrCreateInstallationId],
            @"manifest": EXNullIfNil([[self class] appConfig]),
            @"platform": @{
                @"ios": @{
```

Comment:
the `platform.ios` looks suspicious for macos. maybe we need to use `platform.macos`

Reply:
Its keys are all deprecated (except buildNumber but it's not really platform-specific) and so I think this needs to be removed at all. It's not in scope of this PR.

Reply:
didn't notice most properties are deprecated. it's fine to keep it as-is.